repository/n.json

```diff
@@ -256,6 +256,18 @@
 				}
 			]
 		},
+		{
+			"name": "NeoVintageousToggle",
+			"details": "https://github.com/NeoVintageous/NeoVintageousToggle",
+			"labels": ["neovintageous", "toggle", "vim", "vi", "vintage", "vintageous", "neovim", "nvim", "emulation", "emulator", "editor emulation"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"tags": true
+				}
+			],
+			"previous_names": ["ToggleNeoVintageous"]
```

Comment:
I prefer having the previous names underneath the name, so that it’s still easily seen even if for instance the releases list grows longer.